toolboxes/feel/feelmodels/fluid/fluidmechanicsupdatejacobian.cpp

```diff
@@ -206,7 +206,7 @@ FLUIDMECHANICS_CLASS_TEMPLATE_TYPE::updateJacobian( DataUpdateJacobian & data )
 
     //--------------------------------------------------------------------------------------------------//
     // user-defined additional terms
-    this->updateJacobianAdditional( J, _BuildCstPart );
+    this->updateJacobianAdditional( data );
```

Comment:
the methods updateJacobianAdditional, updateResidualAdditional, updateLinearAdditionalmethod should be removed and replaced by system proposed by the ModelAlgebraicFactory :  addFunctionJacobianAssembly, addFunctionResidualAssembly and addFunctionLinearAssembly. It's used all multiphysics toolboxes and it allows to reduce the intrusion in fluid toolbox

Reply:
Ok, I am looking into this !

Reply:
Done (c.f. f266de9) !